packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -1229,24 +1232,26 @@ async function handler(
 
       // Need to change the new seat reference and attendee record to remove it from the old booking and add it to the new booking
       // https://stackoverflow.com/questions/4980963/database-insert-new-rows-or-update-existing-ones
-      await Promise.all([
-        await prisma.attendee.update({
-          where: {
-            id: seatAttendee.id,
-          },
-          data: {
-            bookingId: newTimeSlotBooking.id,
-          },
-        }),
-        await prisma.bookingSeat.update({
-          where: {
-            id: bookingSeat.id,
-          },
-          data: {
-            bookingId: newTimeSlotBooking.id,
-          },
-        }),
-      ]);
+      if (seatAttendee?.id && bookingSeat?.id) {
```

Comment:
This has wrong type from main, had to do some conditionals to don't run if seatAttendee is null.